development/compiling/cross-compiling_for_ios_on_linux.rst

```diff
@@ -106,7 +120,7 @@ Build cctools:
 
     $ git clone https://github.com/tpoechtrager/cctools-port.git
     $ cd cctools-port/usage_examples/ios_toolchain
-    $ ./build.sh /path/iPhoneOS9.1.sdk.tar.xz arm64
+    $ ./build.sh /path/iPhoneOSX.x.sdk.tar.xz arm64
```

Comment:
IMO these should have a note that users need to replace the `X.x` version with whatever they're using